server/common.ts

```diff
@@ -49,16 +50,16 @@ export let uploadHandler = multer({
 			cb(null!, os.tmpdir());
 		},
 		filename: function (req, file, cb) {
-			cb(null!, `${file.fieldname}-${Date.now()}.${path.extname(file.originalname)}`);
+			cb(null!, `${file.fieldname}-${Date.now()}${path.extname(file.originalname)}`);
```

Comment:
Basically, the `!` operator is the [non-null assertion operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator) that suppresses null checking because we have strict null checking enabled in `tsconfig.json`. Here, this is required because `multer`'s type definitions have a bug where the callback's first argument's type is `Error` and not `Error | null` so it can't accept the not having an error unless `null!` is put to suppress the check.